ui/src/app/styles/cards.css

```diff
@@ -7,3 +7,109 @@
 .card * {
   margin-top: 0rem;
 }
+
+.card.item-card {
+  width: 200px;
+  height: 223px;
+  margin-right: 1rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  cursor: pointer;
+  padding: 0.75rem;
+}
+
+.card.add-card {
+  width: 200px;
+  margin-right: 1rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  cursor: pointer;
+  padding: 0.75rem;
+  height: 105px;
+}
+
+.card.add-card.disabled {
```

Comment:
I think we can combine our techniques here. But specifically I think you should add the "greyed-out" color here, as a further visual cue that this is not clickable.